<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1456</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <property name="spacing">
     <number>0</number>
    </property>

    <!-- Top Horizontal Layout -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_top">
      <item>
       <widget class="QPushButton" name="data_select">
        <property name="text">
         <string>data_select</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="train_button">
        <property name="text">
         <string>start train</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nomal_file"/>
      </item>
      <item>
       <widget class="QLabel" name="malware_file">
        <property name="text">
         <string>No File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Main Content Layout (Tables and Graphics) -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_content">

      <!-- Left Table Layout -->
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_leftTables">
        <item>
         <widget class="QTableWidget" name="data_preprocessor_output"/>
        </item>
        <item>
         <widget class="QTableWidget" name="train_result"/>
        </item>
        <item>
         <widget class="QTableWidget" name="vir_result"/>
        </item>
       </layout>
      </item>

      <!-- Right Graphics Layout -->
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>

     </layout>
    </item>

    <!-- Status Label -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_status">
      <item>
       <widget class="QLabel" name="vir_label"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1456</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
